php/admin-functions-term.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Administration functions for term objects
+ *
+ * @package WP_SEO_Social
+ */
+
+/**
+ * Call printing function for the OG title input for a new term.
+ */
+function wp_seo_social_term_id_to_the_add_term_meta_og_title_input() {
```

Comment:
@kjbenk I see why that's confusing, I'll add a comment. `_add_term` refers to the context though